lib/manager/pub/artifacts.ts

```diff
@@ -0,0 +1,91 @@
+import { TEMPORARY_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import {
+  getSiblingFileName,
+  readLocalFile,
+  writeLocalFile,
+} from '../../util/fs';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+
+async function pubUpdate(
+  pubspecPath: string,
+  newPackageFileContent: string
+): Promise<void> {
+  let cmd = '';
+  if (newPackageFileContent.includes('sdk: flutter')) {
+    cmd = 'flutter pub upgrade';
+  } else {
+    cmd = 'pub upgrade';
+  }
+
+  const execOptions: ExecOptions = {
+    cwdFile: pubspecPath,
+    docker: {
+      image: 'cirrusci/flutter:latest', // Ideally there would be an official image, but this at least gets updated very frequently
```

Comment:
```suggestion
      image: 'cirrusci/flutter:latest', // Ideally there would be an official image, but this at least gets updated very frequently
```

Won't work at all. renovate will add `dockerImageprefix`, so final image will be `docker.io/renovate/cirrusci/flutter` which won't work either 😉  Will provide the required image soon

Reply:
as far as i understood it should appear at https://hub.docker.com/u/renovate ?

Reply:
updated initial suggestion.

you need to add flutter to the buildpack tools, check my comment on the issue for a sample

Reply:
can you give me any hints how to do it? i'm not familiar with the whole renovate and containerbase codebase :see_no_evil:

Reply:
> updated initial suggestion.
> 
> you need to add flutter to the buildpack tools, check my comment on the issue for a sample

☝️ 👉 https://github.com/renovatebot/renovate/issues/6049#issuecomment-1154942893

Reply:
ah thanks i totally missed that :+1: